docs/building-a-form/common-definitions.md

```diff
@@ -65,91 +65,121 @@ A common type-ahead widget that lets a user type in values and narrow down a lon
+}
+```
+Source: [/src/js/definitions/date.js](../../src/js/definitions/date.js)
 
 ### Date range
 
-Two common date fields with validation to ensure they form a valid range.
+Two common date fields with validation to ensure they form a valid range. Call this exported function.
+```js
+import dateRangeUI from 'us-forms-system/lib/js/definitions/dateRange';
 
-- File: [/src/js/definitions/dateRange.js](../../src/js/definitions/dateRange.js)
-- `uiSchema`: Yes
-- `schema`: No
-- Function or object: Function
+uiSchema: {
+  servicePeriod: dateRangeUI('servicePeriod')
+}
+```
+Source: [/src/js/definitions/dateRange.js](../../src/js/definitions/dateRange.js)
 
 ### File upload
 
-The file upload field. Requires a specified endpoint for the upload.
+*Note: This is currently not functional. The file upload field requires a specified endpoint for the upload. Documentation will be updated later.*
 
-- File: [/src/js/definitions/file.js](../../src/js/definitions/file.js)
-- `uiSchema`: Yes
-- `schema`: Yes (the same as in vets-json-schema)
-- Function or object: Function for `uiSchema`, object for `schema`
+Source: [/src/js/definitions/file.js](../../src/js/definitions/file.js)
 
 ### Full name
 
 The normal name field, including first, middle, last, and suffix.
+```js
+import fullNameUI from 'us-forms-system/lib/js/definitions/fullName';
 
-- File: [/src/js/definitions/fullName.js](../../src/js/definitions/fullName.js)
-- `uiSchema`: Yes
-- `schema`: No
-- Function or object: Object
+uiSchema: {
+  fullName: fullNameUI
+}
+```
+Source: [/src/js/definitions/fullName.js](../../src/js/definitions/fullName.js)
 
 ### Month/year
 
-The common date field, excluding day field, with basic validation.
+The common date field, excluding day field, with basic validation. Call this exported function with the label to be displayed on the field.
+```js
+import monthYearUI from 'us-forms-system/lib/js/definitions/monthYear';
 
-- File: [/src/js/definitions/monthYear.js](../../src/js/definitions/monthYear.js)
-- `uiSchema`: Yes
-- `schema`: No
-- Function or object: Function
+uiSchema: {
+  serviceStart: monthYearUI('Month/Year Service Started')
+}
+```
+Source: [/src/js/definitions/monthYear.js](../../src/js/definitions/monthYear.js)
 
 ### Month/year range
 
-Two common date fields, excluding day field, with validation to ensure the dates form a valid range.
+Two common date fields, excluding day field, with validation to ensure the dates form a valid range. Similar to the `Date range` above but without the days. Call this exported function.
+```js
+import monthYearRangeUI from 'us-forms-system/lib/js/definitions/monthYearRange';
 
-- File: [/src/js/definitions/monthYearRange.js](../../src/js/definitions/monthYearRange.js)
-- `uiSchema`: Yes
-- `schema`: No
-- Function or object: Function
+uiSchema: {
+  serviceStart: monthYearRangeUI()
+}
+```
+Source: [/src/js/definitions/monthYearRange.js](../../src/js/definitions/monthYearRange.js)
 
 ### Non-required full name
```

Comment:
This one is supposed to be a `schema` modifier but we don't have the actual schema for `fullName` here do we? The starter app has it copied there rather than importing it from USFS. In any case I wonder if we should delete this for now, it's not that important and could be distracting.

Reply:
Right, the schema for `fullName` doesn't exist in USFS.